packages/flame_texturepacker/lib/flame_texturepacker.dart

```diff
@@ -0,0 +1,75 @@
+library flame_texturepacker;
+
+import 'dart:convert';
+
+import 'package:flame/extensions.dart';
+import 'package:flame/flame.dart';
+import 'package:flame/game.dart';
+import 'package:flame/sprite.dart';
+import 'package:flame_texturepacker/atlas/texture_packer_atlas.dart';
+
+export 'package:flame_texturepacker/atlas/model/texture_packer_sprite.dart';
+
+extension TexturepackerLoader on Game {
+  /// Loads the specified pack file, using the parent directory of the pack file
+  /// to find the page images.
+  Future<TexturePackerAtlas> fromAtlas(String assetsPath) async =>
+      TexturePackerAtlas().load(assetsPath);
+
+  @Deprecated('Please use fromAtlas() and files with extension .atlas')
+  Future<List<Sprite>> fromJSONAtlas(String imagePath, String dataPath) async {
+    final jsonFrames = await _loadJsonFrames(dataPath);
+    final image = await Flame.images.load(imagePath);
+
+    final sprites = jsonFrames.values.map((value) {
+      return _createSprite(value as Map<String, dynamic>, image);
+    });
+
+    return sprites.toList();
+  }
+
+  @Deprecated('Please use fromAtlas() and files with extension .atlas')
+  Future<Map<String, Sprite>> fromJSONAtlasAsMap(
+    String imagePath,
+    String dataPath,
+  ) async {
+    final jsonFrames = await _loadJsonFrames(dataPath);
+    final image = await Flame.images.load(imagePath);
+    final map = <String, Sprite>{};
+
+    jsonFrames.forEach((filename, value) {
+      final id = _filenameWithoutExtension(filename);
+      map[id] = _createSprite(value as Map<String, dynamic>, image);
+    });
+
+    return map;
```

Comment:
oh dartlin how I miss you!

Reply:
Not sure why they didn't just use `map` here, but I removed all deprecated stuff now anyways.

Reply:
oh you could use a map literal right?
on dartlin it would be an `.associate()`